SI/Sem.02/Pract.02/ConsoleApplication_prakt2_12.10.22.cpp

```diff
@@ -0,0 +1,22 @@
+// ConsoleApplication_prakt2_12.10.22.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    int year;
+    cin >> year;
+    if (year > 0 && year % 100 != 0 && year % 4 == 0)

```

Comment:
The condition is not correct 

` For example, in the [Gregorian calendar](https://en.wikipedia.org/wiki/Gregorian_calendar), each leap year has 366 days instead of 365, by [extending February to 29 days](https://en.wikipedia.org/wiki/February_29) rather than the common 28. These extra days occur in each year that is an [integer](https://en.wikipedia.org/wiki/Integer) multiple of 4 (except for years evenly divisible by 100, but not by 400)`